src/pke/include/cryptocontext.h

```diff
@@ -1016,7 +1016,7 @@ class CryptoContextImpl : public Serializable {
             ciphertext->SetEncodingType(plaintext->GetEncodingType());
             ciphertext->SetScalingFactor(plaintext->GetScalingFactor());
             ciphertext->SetScalingFactorInt(plaintext->GetScalingFactorInt());
-            ciphertext->SetDepth(plaintext->GetDepth());
+            ciphertext->SetNoiseScaleDeg(plaintext->GetDepth());
```

Comment:
Why are we changing the ciphertext param depth to noiseScaleDeg but not the plaintext param?

Reply:
yes